packages/rum-core/src/domain/resource/matchRequestTiming.spec.ts

```diff
@@ -59,6 +59,28 @@ describe('matchRequestTiming', () => {
     expect(matchingTiming).toEqual(undefined)
   })
 
+  it('should match all timings to the same request done at the same time', () => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ ‏I am a bit confused by the test description, FMU the implementation avoid to reuse an entry that has already been used for a previous matching.
Do we expect more than that?